app/src/main/java/com/infomaniak/drive/ui/fileList/SelectFolderActivity.kt

```diff
@@ -56,7 +55,7 @@ class SelectFolderActivity : BaseActivity() {
         val driveID = intent.extras?.getInt(USER_DRIVE_ID_TAG) ?: throw MissingFormatArgumentException(USER_DRIVE_ID_TAG)
         val customArgs = intent.extras?.getBundle(CUSTOM_ARGS_TAG)
         val userDrive = UserDrive(userID, driveID)
-        mainViewModel.realm = FileController.getRealmInstance(userDrive)
```

Comment:
Don't remove this

Reply:
No more need because it is managed elsewhere